test/e2e/llmisvc/test_llm_inference_service.py

```
@@ -465,6 +493,45 @@ def delete_llmisvc(kserve_client: KServeClient, llm_isvc: V1alpha1LLMInferenceSe
+
+        if not should_skip:
+            delete_llmisvc(kserve_client, llm_isvc)
+        elif skip_all:
+            print(
+                f"⏭️  Skipping deletion of {service_name} (SKIP_RESOURCE_DELETION=True)"
+            )
+        elif test_failed and skip_on_failure:
+            print(
+                f"⏭️  Skipping deletion of {service_name} due to test failure (SKIP_DELETION_ON_FAILURE=True)"
+            )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)